core/app/models/spree/tax/output/item.rb

```diff
@@ -0,0 +1,10 @@
+module Spree
+  module Tax
+    module Output
+      class Item
+        include ActiveModel::Model
+        attr_accessor :id, :tax_rate, :amount, :included_in_price
```

Comment:
Tax exempt items can just have an adjustment_amount of `0`. We can calculate shipping rates without passing database IDs to external services, so why can we not do the same with taxation? :)